### ISSUE TITLE
[3.0.0-rc1] Blank page when trying to create a Dashboard Widget

### ISSUE THREAD

**Issue by halftrainedharry** [modxcms/revolution#15982]
### Summary
When I try to create a new Dashboard Widget in the manager, no form is shown.

### Step to reproduce
In the manager go to "Dashboards" -> tab "Widgets" -> click the "Create" button. (`manager/?a=system/dashboards/widget/create`)

### Observed behavior
The panel on the right is blank. JS error in the console.

This line in the JS code throws an error, because `Ext.getCmp('modx-extended-form')` is `undefined`. 

https://github.com/modxcms/revolution/blob/d1c747d76906f8437c5f380ece89447a2d76c2d7/manager/assets/modext/widgets/system/modx.panel.dashboard.widget.js#L271

### Possible solution
Check if `Ext.getCmp('modx-extended-form')` is defined before trying to call `.disable()` on it.

```
if (Ext.getCmp('modx-extended-form')){
    Ext.getCmp('modx-extended-form').disable();
}
//or maybe
Ext.getCmp('modx-extended-form')?.disable();
```

### Expected behavior
The form to create a widget is shown.

### Environment
MODX 3.0.0-rc1

**Comment by JoshuaLuckers** [comment]
Would it be possible for you to submit a PR with the proposed solution?

**Comment by Ibochkarev** [comment]
Fixed #15987
